src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/CourtCaseServiceClient.java

```diff
@@ -66,6 +69,24 @@ public Mono<Hearing> getHearing(String hearingId) {
                 });
     }
 
+    public Mono<Hearing> getHearing(String hearingId, String courtCaseId) {
+        final String path = String.format(courtCaseByHearingIdAndCourtCaseIdTemplate, hearingId, courtCaseId);
+
+        // Get the existing case. Not a problem if it's not there. So return a Mono.empty() if it's not
+        return restHelper.get(path)
+            .retrieve()
+            .onStatus(HttpStatusCode::isError, (clientResponse) -> handleGetError(clientResponse, hearingId))
+            .bodyToMono(CCSExtendedHearing.class)
+            .map(response -> {
+                log.debug("GET succeeded for the hearing at {}", path);
```

Comment:
Do we really need to log this?

Reply:
https://mojdt.slack.com/archives/C69NWE339/p1731065730598699

Reply:
also, should this replace the method above which only takes a hearingId?

Reply:
Removed logging. The method with just hearingId is still being used in ReplayHearingService although it might need some thought on how to switch it to use the new one.